types/options.d.ts

```diff
@@ -27,26 +25,29 @@ interface ZoomOptions {
 	drag: boolean | DragEffectOptions;
 
 
-	// Zooming directions. Remove the appropriate direction to disable
-	// Eg. 'y' would only allow zooming in the y direction
-	// A function that is called as the user is zooming and returns the
-	// available directions can also be used:
-	//   mode: function({ chart }) {
-	//     return 'xy';
-	//   },
+	/** 
+	 * Zooming directions. Remove the appropriate direction to disable
+	 * Eg. 'y' would only allow zooming in the y direction
+	 * A function that is called as the user is zooming and returns the
+	 * available directions can also be used:
+	 *    mode: function({ chart }) {
+	 *      return 'xy';
+	 *    },
+	 */	
 	mode: Mode | { (char: Chart): Mode };
 
 	/**
-	  * Format of min zoom range depends on scale type
-	  */
-	rangeMin?: XyValue;
+	 * Format of min zoom range depends on scale type
+	 */
+	rangeMin?: Point;
```

Comment:
I think `number | string` describes these better. Point is `{x, y}`, which is not correct for these.

(So the `XyValue` is also incorrect)

Reply:
Sorry, I used the "README.md" as source

```
rangeMin: {
    // Format of min zoom range depends on scale type
    x: null,
    y: null
   },
```

Reply:
I stand corrected, proper type would be `{x: number | string, y: number | string}`

Reply:
ok, I'm not surce if
```
 const rangeMin = zoomPanOptions.rangeMin[zoomPanOptions.scaleAxes];
    if (newMin < rangeMin) {
```
is meant for a string comparison... but doesn't matter :-)

Thanks for so mutch for your work and hopefully this will be a good starting point for you and others :-)

Reply:
Right, I though it was available for all scales. Seems it only works for numeric scales. It'll get fixed at some point in some direction :)